src/renderer/components/ContextMenu/ContextMenu.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { createContext, useMemo, useState } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+import { clipboard } from 'electron';
+import { useTranslation } from 'react-i18next';
+import { ContextData } from './types';
+import * as Styled from './style';
+
+interface Props {
+  children: React.ReactNode[];
+}
+
+type HandlerFunctionCreator = (
+  data: ContextData
+) => (event: React.MouseEvent) => void;
+
+export const MenuContext = createContext<HandlerFunctionCreator>(
+  () => () => {}
+);
```

Comment:
I would usually make context with `undefined` as a possibility to not create these fake values. Not the biggest inconvenience here but can sometimes be a bother when making context with bigger values.

```suggestion
export const MenuContext = createContext<HandlerFunctionCreator  | undefined>(
  undefined
);
```

Reply:
This works well for catching missing context providers in conjuction with the comment for `src/renderer/components/ContextMenu/useContextMenu.ts`.